src/parser.rs

```diff
@@ -88,7 +95,7 @@ pub fn check_chunk_range(chunks: &Vec<EncodedChunk>) -> Result<(), RestoreError>
         }
         actual_chunk_ids.insert(chunk.id);
     }
-    let expected_chunk_ids: HashSet<u16> = (1..expected_total + 1).collect();
+    let expected_chunk_ids: HashSet<u16> = (1..=expected_total).collect();
```

Comment:
I expected there would be something like that but didn't go looking for it. Good to know. ANy other tricks?

Reply:
I think that's all for the range syntax